modules/lead/toolchain/variables.tf

```diff
@@ -33,6 +33,10 @@ variable "ingress_external_traffic_policy" {
   default = ""
 }
 
+variable "grafeas_version" {
```

Comment:
Can you pull this all the way up into `environment-aws` so that we can specify the version in the lead-environments repo?  Will also need to update the pipeline for the helm chart to trigger a gitops PR when the grafeas version changes.

Reply:
fixed